spec/models/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe User do
+  it { should validate_presence_of(:first_name) }
+  it { should validate_presence_of(:last_name) }
+  it { should validate_inclusion_of(:role)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.